engine/sync_manager.go

```diff
@@ -110,44 +110,15 @@ func (m *syncManager) Start() error {
 
 		if !supportsREST && !supportsWebsocket {
 			log.Warnf(log.SyncMgr,
-				"Loaded exchange %s does not support REST or Websocket.\n",
+				"Loaded exchange %s does not support REST or Websocket.",
 				exchangeName)
 			continue
 		}
 
 		var usingWebsocket bool
 		var usingREST bool
 		if supportsWebsocket && exchanges[x].IsWebsocketEnabled() {
-			ws, err := exchanges[x].GetWebsocket()
-			if err != nil {
-				log.Errorf(log.SyncMgr,
-					"%s failed to get websocket. Err: %s\n",
-					exchangeName,
-					err)
-				usingREST = true
-			}
-
-			if !ws.IsConnected() && !ws.IsConnecting() {
-				if m.websocketDataReceiver.IsRunning() {
-					go m.websocketDataReceiver.WebsocketDataReceiver(ws)
-				}
-
-				err = ws.Connect()
-				if err == nil {
-					err = ws.FlushChannels()
-				}
-				if err != nil {
-					log.Errorf(log.SyncMgr,
-						"%s websocket failed to connect. Err: %s\n",
-						exchangeName,
-						err)
-					usingREST = true
-				} else {
-					usingWebsocket = true
-				}
-			} else {
-				usingWebsocket = true
-			}
+			usingWebsocket = true
```

Comment:
I'd like to see an additional check here to see if the websocket manager is enabled, otherwise we're wasting CPU cycles until this times out per asset which is quite costly if we have hundreds of items